src/lib.rs

```diff
@@ -351,6 +351,19 @@ mod tests {
         let logical = super::get();
         let physical = super::get_physical();
         println!("physical: {:?}, logical: {:?}", physical, logical);
-        assert!(physical <= logical);
+        if let Ok(_) = ::std::env::var("EXPECTED_TRAVIS_CPU_COUNT") {
```

Comment:
Since `get()` checks task affinity, it's understandable that this test could be wrong. Considered that a test env can set `NUM_CPUS_TEST_GET`/`NUM_CPUS_TEST_GET_PHYSICAL`, it seems like perhaps this test should just be deleted. Whatcha think?

Reply:
Agreed. Removed in 506b2af0b50430e2383451fe52d2c7399731e332